config/routes.rb

```diff
@@ -63,18 +63,14 @@
   get "forms", to: "forms#all"
   get "forms/*type", to: "forms#new"
 
-  # controller :collections do
-  #   get "finding_aids/:id" => :finding_aids
-  # end
-  #
-  # If a finding aid can belong to multiple collections, we can't
-  # determine which collection to display as the parent collection
+  controller :collections do
+    get "finding_aids/:id" => :finding_aids
+  end
 
   controller :events do
-    get "events/past" => :index
+    get "events/past" => :past
```

Comment:
If we add a , `as: :past_events` parameter here
```suggestion
    get "events/past" => :past, as: :past_events
```

we can then replace all  calls to the controller and action with "past_events_path". See comments inline above.